cmd/parca-agent/main.go

```diff
@@ -159,7 +160,14 @@ func main() {
 	}
 
 	externalLabels := getExternalLabels(flags.ExternalLabel, flags.Node)
-	tm := target.NewManager(logger, reg, externalLabels, ksymCache, profileListener, debugInfoClient, flags.ProfilingDuration, flags.TempDir)
+	counter := promauto.With(reg).NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "parca_agent_profiler_missing_stacks_total",
+			Help: "Number of missing profile stacks",
+		},
+		[]string{"type"})
+
+	tm := target.NewManager(logger, counter, externalLabels, ksymCache, profileListener, debugInfoClient, flags.ProfilingDuration, flags.TempDir)
 
```

Comment:
Create a metrics struct in `profiler.go` and add a helper function to initialise it maybe.